### DISCUSSION TITLE
'CryptographyDeprecationWarning' message appears while using spacectl

### DISCUSSION THREAD

**Discussion by gikang82** [cloudforet-io/community#19] [General]
Hi ~
I`m using spacectl frequently to check my service.
But this messages below are annoying me.

Messages below appears every spacectl commands
`> spacectl api-resources
/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/jose/backends/cryptography_backend.py:23: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_from_bytes, int_to_bytes`

May be some libraries need to be upgraded.

**Comment by whdalsrnt** [comment]
Hi. @gikang82

This problem is a package version issue of crpytography and will be solved by updating the dependencies of spacectl.

**Comment by whdalsrnt** [comment]
This item is linked in the issue below.
- https://github.com/cloudforet-io/community/issues/20
